R/load-ace-constant-columns.R

```diff
@@ -206,9 +205,24 @@ standardize_ace_column_names <- function(df) {
                   new == "task_switch_state" ~ "taskswitch_state",
                   TRUE ~ new) # for cross compatibility b/w emailed and pulvinar)
   names(df) = new
+
+  # With newer data formats (nexus), "PID" may already be included alongside
+  # an existing "participant_id". In this case, do not map "participant_id"
+  # to "PID" as we'll be creating a duplicate
+  rename_conditionally(df, "participant_id", COL_PID)
+  rename_conditionally(df, "user_id", COL_PID)
```

Comment:
Quick question: Are these actually modifying `df` if they are not assigning the output of `rename_conditionally()` back to `df`? I realize there may yet be R behaviors I'm unaware of, so I figured I'd ask whether this definitely works or whether it ought to be:

```suggestion
  df <- rename_conditionally(df, "participant_id", COL_PID)
  df <- rename_conditionally(df, "user_id", COL_PID)
```

Reply:
I think you're right actually. I verified that `df` is not getting modified despite being passed in. Will update this